worlds/alttp/UnderworldGlitchRules.py

```diff
@@ -15,7 +15,7 @@ def underworld_glitch_connections(world, player):
     specrock.exits.append(kikiskip)
     mire.exits.extend([mire_to_hera, mire_to_swamp])
 
-    if world.worlds[player].fix_fake_world:
+    if world.worlds[player].options.fix_fake_world:
```

Comment:
This needs to be reverted - fix_fake_world is an ALTTPWorld attribute, not an option